recipes.py

```diff
@@ -10,11 +10,16 @@
 CHANNEL_ID = os.getenv('CHANNEL_ID')
 
 if __name__ == "__main__":
-    image_url = f'https://external.api.yle.fi/v1/teletext/images/811/1.png?app_id={APP_ID}&app_key={APP_KEY}'
+    page_number = 811
+    image_url = f'https://external.api.yle.fi/v1/teletext/images/{page_number}/1.png?app_id={APP_ID}&app_key={APP_KEY}'
     print(image_url)
     local_filename = 'recipe_image.jpg'
-    
+
     download_image(image_url, local_filename)
 
-    post_image_to_channel(CHANNEL_ID, local_filename, SLACK_BOT_TOKEN, "Viikon reseptit")
-    post_subpages(811)
+    thread: str | None = None
+    resp = post_image_to_channel(CHANNEL_ID, local_filename, SLACK_BOT_TOKEN, "Viikon reseptit")
+    if resp is not None:
+        thread = resp.get("ts")
```

Comment:
I'm not sure if this is correctly fetched from the `SlackResponse` object. Should be tested before merging